app/controllers/concerns/event_publisher.rb

```diff
@@ -0,0 +1,22 @@
+module EventPublisher
+  # Fires an event using ActiveSupport::Notifications and passes the event_record
+  # payload, action, and user as the event payloads.
+  #
+  # event_name: String
+  # event_record: Eventable model record
+  #
+  # Returns nil
+  def publish(event_name, event_record)
```

Comment:
I like the name in isolation, however seeing it in use in a controller (e.g. `publish 'issue.created', {...}`) left me wondering where is publish coming from. Should we go for the slightly uglier, slightly clearer `publish_event` name?